apps/frontend/src/renderer/components/github-prs/components/PRFilterBar.tsx

```
@@ -0,0 +1,357 @@
+              return (
+                <div
+                  key={item}
+                  className={cn(
+                    "relative flex cursor-pointer select-none items-center rounded-sm px-2 py-2 text-sm outline-none transition-colors hover:bg-accent hover:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+                    isSelected && "bg-accent/50"
+                  )}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    toggleItem(item);
+                  }}
+                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)